### PR TITLE
New Artado Design

### PR DESCRIPTION
**Pull request by ardatdev**
I am trying to implement [this design](https://artado.lareii.repl.co/) to Artado. 
Things to do:
- Center the arrow
- Dark Theme
- Renew the search page
- Make sure this new design doesn't breaks themes in Workshop

Right now, it looks like this:

resim

### CODE REVIEW

**Comment by ardatdev** [comment]
This PR also includes server-side Google results, but they are in testing either.

File: artadosearch/Default.aspx [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <script src="/js/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
     <script src="/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
     <script src="/js/mdb.min.js"></script>
+    <script src="/js/tailwind.js"></script>
```
Comment (Arnolxu) [inline artadosearch/Default.aspx:16]: ah yes, another js library 😍

**Comment by LinuxUsersLinuxMint** [comment]
> I am trying to implement [this design](https://artado.lareii.repl.co/) to Artado. Things to do:
> 
> * Center the arrow
> * Dark Theme
> * Renew the search page
> * Make sure this new design doesn't breaks themes in Workshop
> 
> Right now, it looks like this:
> 
> resim

Whoa great design, I'm so glad. I am proud to work with the Artado team and our goal is to be recognized by the whole world @ardatdev @Arnolxu @abulat189 @LinuxUsersLinuxMint :)

**Comment by LinuxUsersLinuxMint** [comment]
By the way, I am very curious when this design will come. The design looks really good. If this design comes, I think Artado Search will be more known.

**Comment by ardatdev** [comment]
> By the way, I am very curious when this design will come. The design looks really good. If this design comes, I think Artado Search will be more known. @ardatdev

Thanks. No idea when it will be ready for production. There is still things to do and also this update might be more than desing update. I am also trying to add new features, like server-side Google results etc. This update might take weeks or months to merge.

**Comment by LinuxUsersLinuxMint** [comment]
> > By the way, I am very curious when this design will come. The design looks really good. If this design comes, I think Artado Search will be more known. @ardatdev
> 
> Thanks. No idea when it will be ready for production. There is still things to do and also this update might be more than desing update. I am also trying to add new features, like server-side Google results etc. This update might take weeks or months to merge.

Okay, I'm waiting patiently for the new design. It's great to see Artado Search flourishing.

**Comment by ardatdev** [comment]
The search page is completed. Now only thing to do is fix the settings page's night theme and redesign the Workshop.

Right now Artado looks like [this](https://forum.artado.xyz/d/24-artados-fresh-look-check-out-our-new-design)

**Comment by LinuxUsersLinuxMint** [comment]
> The search page is completed. Now only thing to do is fix the settings page's night theme and redesign the Workshop.
> 
> Right now Artado looks like [this](https://forum.artado.xyz/d/24-artados-fresh-look-check-out-our-new-design)

The theme is really excellent, this is great for Artado Search.

**Comment by ardatdev** [comment]
The promotion texts on the home page haven't translated yet, also as I mentioned in the last commit the new design is not responsive. Can someone handle these issues?

**Pull request merged**